homeassistant/components/cast/__init__.py

```diff
@@ -74,7 +74,7 @@ async def async_get_media_browser_root_object(
     async def async_browse_media(
         self,
         hass: HomeAssistant,
-        media_content_type: str,
+        media_content_type: MediaType | str,
```

Comment:
Same for `media_content_type` here:
https://github.com/home-assistant/core/blob/9a5fe950a4895b3bc20664f9d41605998049a1fe/homeassistant/components/cast/media_player.py#L601-L603